internal/services/springcloud/spring_cloud_certificate_resource_test.go

```diff
@@ -145,7 +146,7 @@ data "azurerm_client_config" "current" {
 }
 
 data "azuread_service_principal" "test" {
-  display_name = "Azure Spring Cloud Domain-Management"
+  display_name = "Azure Spring Cloud Resource Provider"
```

Comment:
is that a breaking change in the service side?

Reply:
Hi Tom,

I've confirmed it with service team that both SP can be used to create a certificate. But how to choose the SP is depending on the use case, in this case - `exclude-private_key`, it must use `Azure Spring Cloud Resource Provider`. 

`Azure Spring Cloud Domain-Management` is used in custom domain.


I've updated the example in the document as well.